src/kv/test/kv_test.cpp

```diff
@@ -745,4 +760,74 @@ TEST_CASE("private recovery map swap")
       // impossible during recovery.
     }
   }
+}
+
+TEST_CASE("replicated and derived table serialisation")
```

Comment:
Could this be moved to `kv_serialisation.cpp` which is the unit test file for KV serialisation (also automatically run as part of `kv_test`).

Reply:
Done :)